Dockerfile

```
@@ -1,4 +1,12 @@
 FROM google/dart:2.13
+
+WORKDIR /build
 COPY pubspec.yaml .
+
 RUN dart pub get
+
+COPY /lib ./lib/
+RUN cd lib && tar -czvf ../cdn_assets.tar.gz *.js
```

While looking at the artifact, I noticed this isn't including the source map files. Could we add those?
```suggestion
RUN cd lib && tar -czvf ../cdn_assets.tar.gz *.js *.map
```

---

@greglittlefield-wf done